new.html

```diff
@@ -19,32 +18,32 @@ <h1 id="heading">Font Library</h1>
     </div>
     <div>
 
-<form action="index.html" method="post" enctype="multipart/form-data">
+<form action="index.html" method="post">
 <p><label for="fontname" class="label">Font Name</label>
-  <input type="text" id="fontname" name="fontname" value="" placeholder="e.g. helvectica" style="width:100%;"></p>
+  <input required title="only upper and lowercase letters allowed" pattern="[A-Za-z]" type="text" id="fontname" name="fontname" value="" placeholder="e.g. helvectica"></p>
```

Comment:
regex incorrect matches only one character

Reply:
[A-Za-z0-9-]+

Reply:
Should this do it?

Reply:
Ok, so here you are matching 1 or more upper and lower case characters, numbers and hyphens. Do you think you will need spaces also?